release/release_tests.yaml

```diff
@@ -806,6 +806,316 @@
     timeout: 1800
     script: OMP_NUM_THREADS=64 RAY_ADDRESS= python run_microbenchmark.py
 
+- name: dask_on_ray_10gb_sort
```

Comment:
Can we change the group of microbenchmark and shuffle_10gb to be `core-nightly-test` instead of `Core nightly tests` for consistency?